Modificare criterii de minim pentru Centrul Local

andreiavram opened the pull request.

andreiavram committed:
Modificare criterii de minim pentru Centrul Local

chabinush commented on regulament_pregatit_feedback.txt:
```diff
@@ -497,7 +497,7 @@ Art. 80. Condițiile minime pe care trebuie să le îndeplinească un lider adul
 Art. 81. Cercetașii care depășesc vârsta de 18 ani, nu își desfășoară activitatea ca seniori și nu își asumă nicio responsabilitate educativă, organizatorică sau administrativă în cadrul ONCR, pot opta pentru a deveni alumni.
         	
-Art. 82. Centrul Local este o structură organizatorică formată din cel puțin două unități (minim 5 lideri și 20 de copii sau tineri) a căror activitate este integrată într-o strategie locală comună.
+Art. 82. Centrul Local este o structură organizatorică formată din cel puțin o unitate (minim 3 lideri, minim 20 de copii și / sau tineri, minim 25 de membri în total) a căror activitate este integrată într-o strategie locală comună.
```
În brief nu e menționată partea de min 20 copii, ci doar lideri + total. Eu aș scoate partea aia, că dacă au 18 copii și 7 adulți (de-am avea așa cazuri), eu cred că tot e ok.

chabinush commented on regulament_pregatit_feedback.txt:
```diff
@@ -537,8 +537,9 @@ Art. 87. Procedura de înființare a unui Centru Local este următoarea:
-		a. să cuprindă cel puțin două unități (25 - 30 membri)
-		b. să fi dovedit capacitate organizatorică și de inițiativă, pentru a fi integrat în structura ONCR.
+		a. să cuprindă cel puțin o unitate (minim 2 patrule)
+		b. să aibă minim 25 de membri (minim 3 lider adulți, minim 20 copii / tineri)
```
minim 3 lideri adulți, minim 25 de membri în total

andreiavram committed:
Integrare feedback definitie Centru Local

Scos limitare minim 20 de copii si tineri

andreiavram committed:
Modificare conditii minime pentru Centrul Local

chabinush committed:
revizuire topică nr minim membri CL 

Feedbackul CD a fost că nu e clar dacă cei 3 lideri sunt incluși în cei 25 membri în total și să schimbăm topica doar.
